components/shapediver/stargate/DesktopClientPanel.tsx

```diff
@@ -213,22 +214,81 @@ export default function DesktopClientPanel(props: Props & StyleProps) {
 		NO_CLIENT.value,
 	);
 
-	const refreshClients = useCallback(async (currentClientValue: string) => {
-		setLoading(true);
-		const clients = await getAvailableClients(true);
-		const clientChoices: {value: string; label: string}[] = [
-			NO_CLIENT,
-			...clients.map((client) => ({
-				value: client.id,
-				label: client.clientName,
-				client: client,
-			})),
-		];
-		setAvailableClients(clientChoices);
-		if (!clientChoices.find((c) => c.value === currentClientValue))
-			await selectClient(undefined);
-		setLoading(false);
-	}, []);
+	const connectAttempt = useCallback(
+		async (
+			client: ISdStargateClientModel | undefined,
+		): Promise<boolean> => {
+			await selectClient(client);
+			// Check if connection was successful
+			const currentNetworkStatus =
+				useShapeDiverStoreStargate.getState().networkStatus;
+			if (currentNetworkStatus === NetworkStatus.connected) {
+				notifications.show({
+					message: `Connection to the client "${client?.clientName}" successful.`,
+				});
+				return true;
+			} else {
+				notifications.error({
+					message: `Connection to the client "${client?.clientName}" failed, please check the status of the client and try again.`,
```

Comment:
@mathieu-shapediver I suggest to rephrase the message to _Failed to connect to client XYZ, please check..._
If you agree, please tag @LemToUp